fe/fe-core/src/main/java/com/starrocks/qe/DDLStmtExecutor.java

```diff
@@ -934,6 +934,15 @@ public ShowResultSet visitCreateCatalogStatement(CreateCatalogStmt stmt, Connect
         @Override
         public ShowResultSet visitDropCatalogStatement(DropCatalogStmt stmt, ConnectContext context) {
             ErrorReport.wrapWithRuntimeException(() -> {
+                String catalogName = stmt.getName();
+                if (!context.getGlobalStateMgr().getCatalogMgr().catalogExists(catalogName)) {
+                    if (stmt.isIfExists()) {
+                        LOG.info("drop catalog[{}] which does not exist", catalogName);
```

Comment:
Failed to drop catalog xxxxx

Reply:
I prefer to keep this message to make it consistent with above message style, eg. "drop storage volume[{}] which does not exist"